### PR TITLE
feat(profiling): add txn link

### PR DESCRIPTION
**Pull request by JonasBa**
Make this a link back to the txn

https://user-images.githubusercontent.com/9317857/222308212-bf8684af-b440-49bf-b56f-1aa1522402b2.mp4

### CODE REVIEW

File: static/app/components/profiling/flamegraph/flamegraphDrawer/profileDetails.tsx [hunk 1]
```diff
@@ -47,6 +49,8 @@ function renderValue(
 
 interface ProfileDetailsProps {
   profileGroup: ProfileGroup;
+  projectId: string;
+  transaction: Event | null;
```
Comment (Zylphrex) [inline static/app/components/profiling/flamegraph/flamegraphDrawer/profileDetails.tsx:53]: We should be more specific than just `Event` here

```suggestion
  transaction: EventTransaction | null;
```

File: static/app/components/profiling/flamegraph/flamegraphDrawer/profileDetails.tsx [hunk 2]
```diff
@@ -11,7 +11,9 @@ import {t} from 'sentry/locale';
 import OrganizationsStore from 'sentry/stores/organizationsStore';
 import {useLegacyStore} from 'sentry/stores/useLegacyStore';
 import {space} from 'sentry/styles/space';
+import {Event} from 'sentry/types';
```
Comment (Zylphrex) [inline static/app/components/profiling/flamegraph/flamegraphDrawer/profileDetails.tsx:14]:
```suggestion
import {EventTransaction} from 'sentry/types';
```

File: static/app/views/profiling/profilesProvider.tsx [hunk 3]
```diff
@@ -86,7 +86,8 @@ function ProfileProviderWrapper(props: FlamegraphViewProps): React.ReactElement
   });
 
   const profileTransaction = useSentryEvent<EventTransaction>(
-    params.orgId,
+    // params.orgId does not seem to be set on dev, so provide a fallback
+    params.orgId || organization.slug,
```
Comment (Zylphrex) [inline static/app/views/profiling/profilesProvider.tsx:90]: Can we just always use `organization.slug`?

**Review by Zylphrex — APPROVED** [review]

File: static/app/views/profiling/profilesProvider.tsx (hunk 3 above)
Comment (JonasBa) [reply static/app/views/profiling/profilesProvider.tsx:90]: I dont see a reason why we couldnt

File: static/app/components/profiling/flamegraph/flamegraphDrawer/profileDetails.tsx (hunk 1 above)
Comment (JonasBa) [reply static/app/components/profiling/flamegraph/flamegraphDrawer/profileDetails.tsx]: Had no idea we even had that tbh

**Pull request merged**